database_schemas/cassandra/benchflow.cql

```diff
@@ -233,6 +234,32 @@ CREATE TABLE exp_cpu (
   PRIMARY KEY (experiment_id, container_id)
 );
 
+CREATE TABLE exp_cpu_core (
```

Comment:
@VincenzoFerme That would mean having rows which is essentially nothing but lists for almost all columns. Not sure if this is the best way to do this. Having only one or two columns being lists is one thing, but all of them might not be the best for Cassandra.